src/main/clojure/cemerick/pomegranate/aether.clj

```diff
@@ -170,22 +170,22 @@
 
 (defn- set-policies
   [repo-builder settings]
-  (doto repo-builder
+  (doto ^org.eclipse.aether.repository.RemoteRepository$Builder repo-builder
```

Comment:
Import?

Reply:
`^RemoteRepository$Builder` would fail for clojure 1.7